OpenSim/Simulation/SimulationUtilities.h

```diff
@@ -103,9 +103,55 @@ inline SimTK::State simulate(Model& model,
 
     return state;
 }
-
 /// @}
 
+/** Moving from OpenSim 3.3 (and earlier) to OpenSim 4.0, MotionTypes for
+    Coordinates are now determined strictly by the coordinates' owning Joint.
+    In older models, the MotionType, particularly for CustomJoints, were user-
+    specified. That entailed in some cases, incorrectly labeling a Coordinate
+    as being Rotational, for example, when it is in fact Coupled. Some models
+    that exhibit this issue are the Rajagopal 2015, le6dof9musc models, where
+    the patella Coordinate was user-specified to be Rotational, but the angle
+    of the patella about a Z-axis of the patella body, is a spline function
+    (e.g. coupled function) of the knee_patella Coordinate. Thus, knee_patella
+    no longer represents a Cartesian rotation and is no longer classified as
+    Rotational. Use this utility to remove any unit conversions from Coordinates
+    that were incorrectly labeled as Rotational in the past. For these Coordinates
+    it will undo the incorrect radians to degrees conversion. */
+inline void updateKinematicsFilesForUpdatedModel(const Model& model, 
+                const std::vector<std::string>& filePaths, std::string suffix="")
+{
```

Comment:
Might be good to check if the provided model is older than `30514`.

If the XML document version is `>=30514` or the XML document version doesn't exist, then this function won't correct old columns, and users will be confused when their data still looks wrong. In this condition, we should either throw an exception or give a warning saying that this function won't really do anything.